src/libs/Navigation/AppNavigator/AuthScreens.tsx

```diff
@@ -66,17 +83,17 @@ Onyx.connect({
 
 Onyx.connect({
     key: ONYXKEYS.PERSONAL_DETAILS_LIST,
-    callback: (val) => {
-        if (!val || timezone) {
+    callback: (value) => {
+        if (!value || timezone) {
             return;
         }
 
-        timezone = lodashGet(val, [currentAccountID, 'timezone'], {});
-        const currentTimezone = Intl.DateTimeFormat().resolvedOptions().timeZone;
+        timezone = value?.[currentAccountID ?? -1]?.timezone ?? {};
```

Comment:
Could the object have some value behind `-1`?
```suggestion
        timezone = value?.[currentAccountID]?.timezone ?? {};
```

Reply:
@rezkiy37 I think no, cause `-1` is not used as an account id

Reply:
Okay, so `value[undefined]` returns `undefined` and goes to the other hand of `??`.

Reply:
Yeah, and I guess it makes sense if there is no `currentAccountID`.